feature-group-definitions/messageerror.dist.yml

```diff
@@ -0,0 +1,36 @@
+# Generated from: messageerror.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: messageerror
+description: The `messageerror` event TODO.
+spec:
+  - https://html.spec.whatwg.org/multipage/indices.html#event-messageerror
+  - https://w3c.github.io/ServiceWorker/#eventdef-serviceworkerglobalscope-messageerror
+status:
+  baseline: low
+  baseline_low_date: 2023-03-27
+  support:
+    chrome: "80"
+    chrome_android: "80"
+    edge: "80"
+    firefox: "65"
+    firefox_android: "65"
+    safari: "16.4"
+    safari_ios: "16.4"
+compat_features:
+  # BroadcastChannel as a whole wasn't supported in Edge 18.
+  #- api.BroadcastChannel.messageerror_event
+  - api.DedicatedWorkerGlobalScope.messageerror_event
+  - api.MessagePort.messageerror_event
+  - api.ServiceWorkerContainer.messageerror_event
```

Comment:
Including this increases the Chrome and Firefox versions, and it's not essential for _every_ use of the messageerror event. But I find it hard to argue the feature is complete without it. We could consider partial implementation before this point.